lib/accordion/index.js

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { themr } from 'react-css-themr';
+import { FaAngleRight } from 'react-icons/fa';
+import classnames from 'classnames';
+import defaultTheme from './theme.module.scss';
+import '../globals/fonts.scss';
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      expandedKeys: [],
+    };
+    this.expandedContentRef = {};
+  }
+
+  toggle = (keyValue) => {
+    const { expandedKeys } = this.state;
+    const { allowMultipleExpanded } = this.props;
+    let updatedExpandedKeys = [...expandedKeys];
+    if (allowMultipleExpanded) {
+      if (updatedExpandedKeys.includes(keyValue)) {
+        updatedExpandedKeys = updatedExpandedKeys.filter(key => key !== keyValue);
+      } else {
+        updatedExpandedKeys.push(keyValue);
+      }
+    } else {
+      updatedExpandedKeys = expandedKeys.includes(keyValue) ? [] : [keyValue];
+    }
+    this.setState({ expandedKeys: updatedExpandedKeys });
+
+    if (this.expandedContentRef[keyValue].clientHeight) {
+      this.expandedContentRef[keyValue].style.height = 0;
+    } else {
+      const wrapper = this.expandedContentRef[keyValue].firstChild;
+      this.expandedContentRef[keyValue].style.height = `${
+        wrapper.clientHeight
+      }px`;
+    }
+  };
+
+  renderContent = () => {
+    const {
+      theme,
+      headerKey,
+      contentKey,
+      borderless,
+      headerClassName,
+      contentClassName,
+      data,
+    } = this.props;
+    const { expandedKeys } = this.state;
+    const iconProps = {
+      className: classnames(theme.icon, theme['expand-icon'], theme.open),
+    };
+    return (
+      <div>
+        {data
+          && data.map(item => (
+            <div
+              id="accordion-list"
+              key={item.key}
+              className={classnames(theme.itemWrapper, {
+                [theme.noBorder]: borderless,
+              })}
+            >
+              <div className={classnames(theme.item)}>
+                {/*  eslint-disable jsx-a11y/no-static-element-interactions */
+                /* eslint-disable jsx-a11y/click-events-have-key-events  */}
+                <div
+                  id="accordion-heading"
+                  className={classnames(
+                    theme.heading,
+                    { [theme.headerActive]: expandedKeys.includes(item.key) },
+                    headerClassName,
+                  )}
+                  onClick={() => this.toggle(item.key)}
+                >
+                  <FaAngleRight
+                    {...iconProps}
+                    className={classnames(
+                      expandedKeys.includes(item.key) ? theme.open : theme.close,
```

Comment:
use the variable here and all other occurrences of `expandedKeys.includes(item.key)` below

Reply:
solved! https://github.com/Codebrahma/react-lite-ui/pull/327/commits/c9650faa1b029d68df1ed882c82d4d43d8576d21